extensions/interactions/TextInput/TextInputValidationService.js

```diff
@@ -17,12 +17,52 @@
  */
 
 oppia.factory('TextInputValidationService', [
-  'baseInteractionValidationService',
-  function(baseInteractionValidationService) {
+  'INTERACTION_SPECS', 'WARNING_TYPES', 'baseInteractionValidationService',
+  function(INTERACTION_SPECS, WARNING_TYPES, baseInteractionValidationService) {
     return {
       getCustomizationArgsWarnings: function(customizationArgs) {
-        // TODO(juansaba): Implement customization args validations.
-        return [];
+        var warningsList = [];
+        baseInteractionValidationService.requireCustomizationArguments(
+          customizationArgs,
+          ['placeholder', 'rows']);
+
+        var placeholder = customizationArgs.placeholder.value;
+        if (!angular.isString(placeholder)) {
+          warningsList.push({
+            type: WARNING_TYPES.ERROR,
+            message: (
+              'Placeholder text must be a string.')
+          });
+        }
+
+        var isInt = function(n) {
+          return angular.isNumber(n) && n % 1 === 0;
+        }
+
+        var rows = customizationArgs.rows.value;
+        if (isInt(rows)) {
+          var textSpecs = INTERACTION_SPECS.TextInput;
+          var customArgSpecs = textSpecs.customization_arg_specs;
+          var rowsSpecs = customArgSpecs[1];
+          var MIN_ROWS = rowsSpecs.schema.validators[0].min_value;
```

Comment:
I wouldn't capitalize this; it's being defined off a variable so it's not a constant.